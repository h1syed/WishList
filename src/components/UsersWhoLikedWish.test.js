import React from "react";
import { shallow, mount  } from "enzyme";
import UsersWhoLikedWish from "./UsersWhoLikedWish";

describe("UsersWhoLikedWish", () => {
    const userswholikedwish = shallow(<UsersWhoLikedWish data={[""]} />);

    it("renders properly", () => {
        expect(userswholikedwish).toMatchSnapshot();
    });
});