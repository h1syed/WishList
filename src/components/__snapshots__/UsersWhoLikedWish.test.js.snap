// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UsersWhoLikedWish renders properly 1`] = `ShallowWrapper {}`;
